### PR TITLE
#1153 Expose bridge

### PR DESCRIPTION
**Pull request by agrimmer**
Provide a new endpoint `configure/bridge/expose` in the API for exposing the bridge.

### CODE REVIEW

File: api/go.mod [hunk 1]
```diff
@@ -16,7 +16,14 @@ require (
 	github.com/gorilla/websocket v1.4.1
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/keptn/go-utils v0.6.1-compat
+	github.com/keptn/keptn/configuration-service v0.0.0-20200406090046-e6e7da3c4531
```
Comment (christian-kreuzberger-dtx) [inline api/go.mod:19]: Was this added on purpose?

Comment (agrimmer) [reply api/go.mod]: Thank you! No this was not intended.

File: api/handlers/configure.go [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+	}
+
+	for _, ns := range nsList.Items {
+		if ns.Name == "istio-system" {
+			return istio, nil
+		} else if ns.Name == "ingress-nginx" {
+			return nginx, nil
+		}
+	}
+	return istio, errors.New("Cannot obtain type of ingress.")
```
Comment (christian-kreuzberger-dtx) [inline api/handlers/configure.go:64]: Are you returning `istio` on purpose here? Or should this be nil?

**Comment by christian-kreuzberger-dtx** [comment]
One more thing that just came up in my mind is whether this works with OpenShift or not.

File: api/handlers/configure.go (hunk 2 above)
Comment (agrimmer) [reply api/handlers/configure.go:64]: The type `Ingress` cannot be nil.

**Comment by agrimmer** [comment]
Good point! I will implement `oc create route edge bridge --service=bridge --port=http --insecure-policy='None' -n keptn --hostname="bridge.keptn.$BASE_URL"` in the `openshift-route-service`.

**Comment by agrimmer** [comment]
Now, this PR also supports exposing the Bridge using OC routes.

**Pull request merged**